Program.cs

```diff
@@ -1,2 +1,22 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using System;
+
+namespace InventoryManagementSystem
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Inventory inventory = new Inventory();
+
+            inventory.AddProduct("Product 11", 10.99, 100);
+            inventory.AddProduct("Product 22", 12.99, 120);
+            inventory.AddProduct("Product 33", 13.99, 130);
+            inventory.ViewAllProducts();
+            inventory.EditProduct("Product 11");
+            inventory.DeleteProduct("Product 22");
+            inventory.SearchForProduct("Product 33");
+            inventory.Exit();
```

Comment:
the called function here is good for testing purposes, But related to the expected outcomes of the task, we need to system with user input/output, so I expected to see the user add the products and choose the operation he want to do, I know you already use readLine in other areas but the whole system not depending on user options